### ISSUE TITLE
How many GPUS are used when training 

### ISSUE THREAD

**Issue by jixiang2016** [swz30/Restormer#15]
Hi, could tell me the detailed information about the GPUS(GPU type and number) used in the experiment and the loss for motion deblur? I do not find these details in the paper, thanks so much.

**Comment by swz30** [comment]
Hi @jixiang2016 

We have released the training codes. By default, the training script uses 8 NVIDIA Tesla V100 GPUs. To use any other number of GPUs, modify [Restormer/train.sh](https://github.com/swz30/Restormer/blob/main/train.sh) and the YAML file corresponding to each task.

**Comment by zaidilyasmughal89** [comment]
Hi authors! Very impressive work! 


Could you kindly elaborate the training time it took for deraining task using 8 V100 GPUs?

Secondly, V100 comes in 16 and 32GB versions. Which version you used?

Regards!
